.github/ISSUE_TEMPLATE/meeting-minutes.md

```diff
@@ -0,0 +1,47 @@
+---
+name: "\U0001F91D Technical Steering Committee Meeting Minutes"
+about: To track Technical Steering Committee meeting agenda and attendance
+title: DD MMM YYYY - Technical Steering Committee Meeting Minutes
```

Comment:
change date format to YYYY MM-MMM DD to make it ordered?

Reply:
Hi @jonfreedman Thanks for the feedback I've implemented your suggestions. In this case im proposing YYYY MMM DD is that okay?